components/cart/add-to-cart.tsx

```diff
@@ -50,15 +50,16 @@ export function AddToCart({
         });
       }}
       className={clsx(
-        'flex w-full items-center justify-center bg-black p-4 text-sm uppercase tracking-wide text-white opacity-90 hover:opacity-100 dark:bg-white dark:text-black',
+        'relative flex w-full items-center justify-center rounded-full bg-blue-600 p-4 tracking-wide text-white hover:opacity-90',
         {
           'cursor-not-allowed opacity-60': !availableForSale,
           'cursor-not-allowed': isPending
         }
       )}
     >
+      <div className="absolute left-0 ml-4">{!isPending ? <PlusIcon className="h-5" /> : null}</div>
```

Comment:
What if, instead of the `PlusIcon` and `LoadingDots` having separate spaces, they occupy the same space? So... 

```
<div className="absolute left-0 ml-4">{isPending ? <LoadingDots className="bg-white" /> : <PlusIcon className="h-5" />}</div>
```